src/PIL/JpegImagePlugin.py

```diff
@@ -409,7 +409,8 @@ def draft(self, mode, size):
             return
 
         d, e, o, a = self.tile[0]
-        scale = 0
+        scale = 1
```

Comment:
Both values are not handled: https://github.com/python-pillow/Pillow/blob/master/src/libImaging/JpegDecode.c#L238

While 1 is more correct.